packages/react-components/react-combobox/library/src/components/Combobox/useCombobox.tsx

```diff
@@ -112,6 +115,18 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
     ...baseState,
   };
 
+  const { targetDocument } = useFluent();
+
+  useOnClickOutside({
+    element: targetDocument,
+    callback: event => {
+      event.preventDefault();
+      setOpen(event as unknown as ComboboxOpenEvents, false);
```

Comment:
Is this typecast necessary? The event param that the `callback` exposes has a `MouseEvent` type and that should be compatible with `ComboboxOpenEvents`, no?

Reply:
Fair enough. :)